tez-api/src/test/java/org/apache/tez/common/TestTezCommonUtils.java

```diff
@@ -413,4 +414,16 @@ public void testGetDAGSessionTimeout() {
   }
 
 
+  @Test
+  public void testMkDirForAM() throws IOException {
+    Configuration remoteConf = new Configuration();
+    remoteConf.set(MiniDFSCluster.HDFS_MINIDFS_BASEDIR, TEST_ROOT_DIR);
+    remoteConf.set(CommonConfigurationKeys.FS_PERMISSIONS_UMASK_KEY, "777");
+    MiniDFSCluster miniDFS = new MiniDFSCluster.Builder(remoteConf).numDataNodes(3).format(true).racks(null)
```

Comment:
unit test works as expected, this patch is so close, 1 more thing, could you please shutdown miniDFS cluster?

Reply:
Hi @abstractdog .Thanks for your suggestion, In this patch, I closed miniDFS after I ran out of it. Thank you for reviewing my code, it has taught me a lot.

Reply:
merged to master, thanks @skysiders for this patch, your tireless work, and quick responses!